From: Alexis Murzeau <amubtdx@gmail.com>
Date: Thu, 8 Mar 2018 23:40:45 +0100
Subject: Avoid use of non-ASCII in plugins

Plugins get loaded in streamlink/session.py with imp.load_module.
imp.load_module uses the system locale to read the given python file.

The only way to ensure this will never fail whatever the locale the user
uses is to either:
 - Avoid imp.load_module
 - Use only ASCII characters in plugins

imp.load_module might be required to support both python 2 and 3 (which
upstream need).
So replace the non ASCII character with an ASCII one.

Forwarded: https://github.com/streamlink/streamlink/pull/1541
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 src/streamlink/plugins/dogan.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/streamlink/plugins/dogan.py b/src/streamlink/plugins/dogan.py
index 90c70ea..a9600d4 100644
--- a/src/streamlink/plugins/dogan.py
+++ b/src/streamlink/plugins/dogan.py
@@ -12,7 +12,7 @@ from streamlink.stream import HLSStream
 
 class Dogan(Plugin):
     """
-    Support for the live streams from DoÄŸan Media Group channels
+    Support for the live streams from Dogan Media Group channels
     """
     url_re = re.compile(r"""
         https?://(?:www.)?
